### ISSUE TITLE
Version "22.04" of Ubuntu is not supported

### ISSUE THREAD

**Issue by chrstphfrtz** [swift-actions/setup-swift#443]
**Describe the bug**

While running a GitHub Action with ```ubuntu-22.04``` the following error occurs

```
Run swift-actions/setup-swift@v1
[8](https://github.com/chrstphfrtz/Dolphin/runs/7598794433?check_suite_focus=true#step:3:9)
Error: Unexpected error, unable to continue. Please report at https://github.com/swift-actions/setup-swift/issues
[9](https://github.com/chrstphfrtz/Dolphin/runs/7598794433?check_suite_focus=true#step:3:10)
Version "22.04" of Ubuntu is not supported
[10](https://github.com/chrstphfrtz/Dolphin/runs/7598794433?check_suite_focus=true#step:3:11)
Stacktrace:
[11](https://github.com/chrstphfrtz/Dolphin/runs/7598794433?check_suite_focus=true#step:3:12)
Error: Version "22.04" of Ubuntu is not supported
[12](https://github.com/chrstphfrtz/Dolphin/runs/7598794433?check_suite_focus=true#step:3:13)
    at Object.<anonymous> (/home/runner/work/_actions/swift-actions/setup-swift/v1/dist/index.js:9091:19)
[13](https://github.com/chrstphfrtz/Dolphin/runs/7598794433?check_suite_focus=true#step:3:14)
    at Generator.next (<anonymous>)
[14](https://github.com/chrstphfrtz/Dolphin/runs/7598794433?check_suite_focus=true#step:3:15)
    at fulfilled (/home/runner/work/_actions/swift-actions/setup-swift/v1/dist/index.js:9044:58)
```

I guess this is intended as ```ubuntu-22.04``` is in beta right now (https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners#supported-runners-and-hardware-resources) but what I am wondering is why the check is even performed. 

Looking at https://github.com/swift-actions/setup-swift/blob/main/src/os.ts the macOS system version is completely ignored as it is set to ```version: "latest"``` as soon as it detects the ```darwin``` os but the linux case specifically checks the Ubuntu version. Therefore, ```macos-12``` works in GitHub Actions although it is not in the list.

This brings me to my question: Is the system version check for Ubuntu needed and if yes, should it then also be implemented for the macOS system version? I am more than happy to help and can open a PR if this is decided.

**Workflow configuration (please complete the following information):**
 - Platform (`runs-on`): [ubuntu-22.04, macos-12]
 - Swift version (`swift-version`): 5.6.1

**Comment by fwal** [comment]
Hi @chrstphfrtz, thank you for your report!

If I remembered correctly I had issues with the version detection on macOS and that's why it was set to always be latest 😅 
Imo it should do version checks for both.

As for the issue with 22.04 specifically I'll add it right away and test it out 👍🏻

**Comment by fwal** [comment]
Thanks to @ffried this is now fixed in version 1.18.0 🎉
